contracts/zero-ex/contracts/src/features/IMetaTransactionsFeature.sol

```diff
@@ -88,15 +105,10 @@ interface IMetaTransactionsFeature {
 
     /// @dev Execute a meta-transaction via `sender`. Privileged variant.
     ///      Only callable from within.
-    /// @param sender Who is executing the meta-transaction..
-    /// @param mtx The meta-transaction.
-    /// @param signature The signature by `mtx.signer`.
+    /// @param state The `ExecuteState` for this metatransaction, with `sender`,
+    ///              `hash`, `mtx`, and `signature` fields filled.
     /// @return returnResult The ABI-encoded result of the underlying call.
-    function _executeMetaTransaction(
-        address sender,
-        MetaTransactionData calldata mtx,
-        bytes calldata signature
-    )
+    function _executeMetaTransaction(ExecuteState memory state)
```

Comment:
This function is intended to be called from outside this feature so it needs to take params like
```
address sender,
MetaTransactionData calldata mtx,
LibSignature.Signature calldata sigature
```

Reply:
I don't understand why.

Reply:
`ExecuteState` has a bunch of fields that outsiders shouldn't have to concern themselves with.